src/routes/DigitalAssets/modules/index.js

```diff
@@ -0,0 +1,22 @@
+// @flow
+
+import {
+  type DigitalAssetsActions,
+} from './digitalAssets'
+
+import {
+  type DigitalAssetsGridActions,
+} from '../routes/Grid/modules/digitalAssetsGrid'
+
+import {
+  type AddAssetActions,
+} from '../routes/AddAsset/modules/addAsset'
+
+import {
+  type EditAssetActions,
+} from '../routes/EditAsset/modules/editAsset'
+
+export type DigitalAssetsAllActions = DigitalAssetsActions |
```

Comment:
the same comment about enum type

Reply:
it could be named, e.g. `DigitalAssetsModuleAction` or `DigitalAssetsAnyAction`, etc